### PR TITLE
Accumulator: add terraform EMR description.

### PR DESCRIPTION
**Pull request by NinjaTrappeur**
Adding a EMR terraform deployment in charge running the Hadoop services.

We're spinning a VPC with two subnetworks: a public one and a private one. Public IP addresses are allocated to the EC2 machines created on the public network. Private adresses are allocated to the EC2 instances created on the private subnetwork. Gateway-wise, we NAT the egress coming from the private network, we just forward the egress coming out of the public sub-network.

The Sequencer is allocated on the public network, the Accumulator on the private network. The Sequencer is able to connect to the master Hadoop node through the VPC.

We're missing the sequencer here, I'm still not 100% sure how to run it (podman on EC2?) and how to connect it to the EMR cluster (likely through a IAM policy?).

### CODE REVIEW

File: accumulator/infra/main.tf [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    # TODO: set up autoscaling policy w/ spot instances.
+    ebs_config {
+      size                 = "20"
+      type                 = "gp2"
+      volumes_per_instance = 1
+    }
+  }
+
+  bootstrap_action {
+    path = "s3://elasticmapreduce/bootstrap-actions/run-if"
```
Comment (ycryptx) [inline accumulator/infra/main.tf:296]: how did you you choose this bootstrap action script?

File: accumulator/infra/main.tf [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+variable "project" {
+  description = "Project name exposed in AWS user tag"
+  default = "mina"
+}
+
+terraform {
+  backend "s3" {
+    bucket = "mina-tf-state"
+    key = "tfstate"
+    region = "eu-central-1"
```
[the remark this answers is not in the document]
Comment (NinjaTrappeur) [reply accumulator/infra/main.tf:15]: Nope, you can't use any input variable/state variables in the provider section.

File: accumulator/infra/main.tf (hunk 1 above)
Comment (NinjaTrappeur) [reply accumulator/infra/main.tf:296]: This is a TODO, that's the EMR hello world bootstrap action.

We'll use this script to install the streaming JARS/NPM dependencies.

Comment (NinjaTrappeur) [reply accumulator/infra/main.tf:296]: * Unless we come up with a pre-bootstrapped custom AMI. That's usually preferable: it drastically decrease the boot time for the autoscaling workers.

**Pull request merged**